test/mocks/adobe.ts

```diff
@@ -367,4 +367,8 @@ export const basicAppMeasurement: AppMeasurement = {
   eVar50: 'cca-1234',
   eVar60: '21.95',
   prop1: 'USD',
+  prop10: 'prop-10',
+  prop20: 'prop-20',
+  prop50: 'prop-50',
+  prop60: 'prop-60',
```

Comment:
Since the ruleset served with data layer capture integrations also sends `prop*` values to `FS.event`, I filled out some test data to equivalently test `eVar*` and `prop*` values.